### PR TITLE
move firstPublished and lastUpdated to common

### PR DESCRIPTION
**Pull request by marjisound**
## What does this change?
This PR moved the `FirstPublished` & `LastUpdated` components to common-rendering. 
## Why?
In order to share these components between AR & DCR

### CODE REVIEW

File: common-rendering/src/components/FirstPublished.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}: {
+	firstPublished: number;
+	blockLink: string;
+}) => {
+	const publishedDate = new Date(firstPublished);
+	return (
+		<a
+			href={blockLink}
+			data-ignore="global-link-styling"
+			// title={publishedDate.toLocaleString()}
```
Comment (DavidLawes) [inline common-rendering/src/components/FirstPublished.tsx:21]: tiny q: I can see we had this same comment in liveblock too, do we know if we need to retain it?

**Review by DavidLawes — APPROVED** [review]
👍

File: common-rendering/src/components/FirstPublished.tsx (hunk 1 above)
Comment (marjisound) [reply common-rendering/src/components/FirstPublished.tsx:21]: Good question. @lucymonie do you know why this line was commented out?

Comment (lucymonie) [reply common-rendering/src/components/FirstPublished.tsx:21]: I don't know @marjisound, but I think it can come out as it's not in use.

**Pull request merged**